src/modules/app/Permissions.js

```diff
@@ -24,5 +26,5 @@ export const requestLocationPermission = async (): RESULTS => {
 
 export const requestPushNotificationPermission = async (): RESULTS => {
   // TODO NATIVE-399 Really request push notification permissions
```

Comment:
```suggestion
  // TODO NATIVE-399 Really request push notification permissions
```

399 is already rejected.

Reply:
Sorry, I did not get the point.

Reply:
Should probably be:
```suggestion
```